src/main/kotlin/snyk/common/lsp/SnykLanguageClient.kt

```diff
@@ -216,8 +267,8 @@ class SnykLanguageClient :
      * Get all the scan publishers for the given scan. As the folder path could apply to different projects
      * containing that content root, we need to notify all of them.
      */
-    private fun getScanPublishersFor(snykScan: SnykScanParams): Set<Pair<Project, SnykScanListenerLS>> =
-        getProjectsForFolderPath(snykScan.folderPath)
+    private fun getScanPublishersFor(folderPath: String): Set<Pair<Project, SnykScanListenerLS>> =
```

Comment:
this should probably be `path: String`, as it could be files or folders

Reply:
Fixed!

Reply:
it's still "folderPath"